### ISSUE TITLE
Incorrect environment variable definition causes unhealthy xconnect container

### ISSUE THREAD

**Issue by DotTech** [Sitecore/Sitecore.Demo.Platform#209]
https://github.com/Sitecore/Sitecore.Demo.Platform/blob/eb63efc91df444d43a110af7321083c6d2f2634f/docker-compose.override.yml#L113

This causes an exception which results in an unhealthy container.
The line should be:

`SITECORE_LICENSE_LOCATION: c:\license\`

**Comment by jflheureux** [comment]
You are right. This is an issue when your host license file is not in the `C:\license` folder.
